domains/mlemingcapoo.json

```diff
@@ -0,0 +1,13 @@
+{
+    "owner": {
+        "username": "mlemingcapoo",
+        "email": "ducvu73@gmail.com"
+    },
+    "record": {
+        "A": ["217.174.245.249"],
+        "AAAA": ["2a00:da00:1800:83a4::1"],
+        "MX": ["hosts.is-a.dev"],
+        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all"
+        "CNAME": "mlemingcapoo.github.io"
```

Comment:
Cannot have CNAME in conjunction with other records and vise versa.

Reply:
so are you suggesting that in the "record" i should've specified "CNAME" only?